### ISSUE TITLE
Missing data in the datamodel

### ISSUE THREAD

**Issue by odomanov** [plk/biblatex#404]
This is not really a bug, since it doesn't seem to affect the output. 
With this MWE
````latex
\RequirePackage{filecontents}
\begin{filecontents}{\jobname.bib}
@XDATA{xd,
  location     = {Loc},
  publisher    = {publisher},
  language     = {russian},
  langid       = {russian},
}
@MVPROCEEDINGS{m1,
 title = {Title1},
 editor = {Editor},
 editortype = {compiler},
 year = 2000,
}
@COLLECTION{m2,
 title = {Title2},
 editor = {Editor},
 editortype = {compiler},
 year = 2000,
}
\end{filecontents}
\documentclass{article}
      
\usepackage[backend=biber]{biblatex}
\addbibresource{\jobname.bib}

\begin{document}
\nocite{*}
\printbibliography
\end{document}
````
running `biber --validate-datamodel` gives warnings:
````
WARN - Datamodel: Entry 'xd' (mvproceedings.bib): Invalid field 'language' for entrytype 'xdata'
WARN - Datamodel: Entry 'xd' (mvproceedings.bib): Invalid field 'location' for entrytype 'xdata'
WARN - Datamodel: Entry 'xd' (mvproceedings.bib): Invalid field 'publisher' for entrytype 'xdata'
WARN - Datamodel: Entry 'm1' (mvproceedings.bib): Invalid entry type 'mvproceedings' - defaulting to 'misc'
WARN - Datamodel: Entry 'm1' (mvproceedings.bib): Invalid field 'editortype' for entrytype 'misc'
WARN - Datamodel: Entry 'm2' (mvproceedings.bib): Invalid field 'editortype' for entrytype 'collection'
````
Some of them should probably be fixed.

**Comment by plk** [comment]
Should be fixed in latest biblatex 3.4/biber 2.5 dev versions.

**Comment by odomanov** [comment]
Works fine. Thank you.
